### DISCUSSION TITLE
Don't know how to handle 'brew doctor' output

### DISCUSSION THREAD

**Discussion by kernel-panic-enjoyer** [Homebrew/discussions#759] [Everyday usage] [answered]
So when I type `brew doctor` I get the following output:

```
❯ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /opt/local/bin/ncursesw6-config
  /opt/local/bin/ncurses6-config
```
Thing is, I dont have any clue where the ncurses config files stem from and I dont want to just go ahead and `rm -rf` them. I tried finding what stuff depends on ncurses but I must've uninstalled whatever that was:
```
❯ brew deps --tree ncurses
ncurses
```
Also, just in case it matters, I've installed libcryptopp via port in the past but removed it since.

**Comment by carlocab** [comment, marked as answer]
Is it actually giving you problems, or are you just bothered by the warnings because they're there? If it's the latter, you can just ignore them:

>```
>Please note that these warnings are just used to help the Homebrew maintainers
>with debugging if you file an issue. If everything you use Homebrew for is
>working fine: please don't worry or file an issue; just ignore this. Thanks!
>```

If it's the former, then `/opt/local` is usually a MacPorts prefix. You must have installed `ncurses` using MacPorts before.

**Reply by kernel-panic-enjoyer** [reply]
You're right! Stupid me. It was a leftover dependency from ports that wasn't cleaned.
